chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/subscriptions/subjects/formatter/RwtPlansUrlChangedFormatter.java

```diff
@@ -0,0 +1,60 @@
+package gov.healthit.chpl.scheduler.job.subscriptions.subjects.formatter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.activity.history.ListingActivityUtil;
+import gov.healthit.chpl.dao.ActivityDAO;
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.dto.ActivityDTO;
+import gov.healthit.chpl.subscription.domain.SubscriptionObservation;
+import gov.healthit.chpl.util.DateUtil;
+import lombok.extern.log4j.Log4j2;
+
+@Component
+@Log4j2(topic = "subscriptionObservationsNotificationJobLogger")
+public class RwtPlansUrlChangedFormatter extends ObservationSubjectFormatter {
+    private static final String DESCRIPTION_REMOVED = "RWT Plans URL %s was removed.";
+    private static final String DESCRIPTION_ADDED_UNFORMATTED = "RWT Plans URL %s was added.";
+    private static final String DESCRIPTION_UPDATED_UNFORMATTED = "RWT Plans URL was changed from %s to %s";
```

Comment:
The first two text lines end with a period; the last one doesn't. I don't know what the other observation strings look like, but we should be consistent across all of them, including the three pre-existing ones